_posts/2017-10-28-service-centre-ui.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
The annotations in the images seem not to be clear enough. Can you please use another font or use a bigger size  for the annotations?